### ISSUE TITLE
devspace deploy fails while kubectl apply works

### ISSUE THREAD

**Issue by fg91** [loft-sh/devspace#1314]
**What happened?** 

I have the following deployments in my `devspace.yaml`:

```yaml
- name: knative-eventing-crd
  namespace: knative-eventing
  kubectl:
    manifests:
    - https://github.com/knative/eventing/releases/download/v0.19.0/eventing-crds.yaml
- name: knative-eventing-infra
  namespace: knative-eventing
  kubectl:
    manifests:
    - https://github.com/knative/eventing/releases/download/v0.19.0/eventing-core.yaml
```

`devspace deploy --deployments=knative-eventing-crd,knative-eventing-infra` gives the following error:

```
[done] √ Successfully deployed knative-eventing-crd with kubectl
[fatal]  Error deploying knative-eventing-infra: failed to unmarshal manifest: error unmarshaling JSON: Object 'Kind' is missing in '{"aggregationRule":{"clusterRoleSelectors":[{"matchLabels":{"duck.knative.dev/addressable":"true"}}]}}'
Please make sure `kubectl apply` does work locally with manifest `https://github.com/knative/eventing/releases/download/v0.19.0/eventing-core.yaml`
```

**However, the following works without a problem:**

```
kubectl apply -f https://github.com/knative/eventing/releases/download/v0.19.0/eventing-crds.yaml
kubectl apply -f https://github.com/knative/eventing/releases/download/v0.19.0/eventing-core.yaml
```

**What did you expect to happen instead?** 
I expected `devspace deploy` to work, just as `kubectl apply`.

**How can we reproduce the bug?** (as minimally and precisely as possible)  
See above.

**Local Environment:**  
- DevSpace Version: devspace version 5.7.5
- Operating System: mac
- Deployment method: kubectl apply

**Kubernetes Cluster:**  
- Cloud Provider: google
- Kubernetes Version: 

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.14-gke.1600", GitCommit:"7c407f5cc8632f9af5a2657f220963aa7f1c46e7", GitTreeState:"clean", BuildDate:"2020-12-07T09:22:27Z", GoVersion:"go1.13.15b4", Compiler:"gc", Platform:"linux/amd64"}
```

**Anything else we need to know?**  

/kind bug

**Comment by FabianKramm** [comment]
@fg91 thanks for reporting this issue! Seems like there is an issue parsing these manifests inside devspace when the `apiVersion` and `kind` are switched, we'll investigate that and fix it for the next release!

**Comment by fg91** [comment]
Ok awesome, thank you for the quick reply! Awesome tool btw, love it.

**Comment by FabianKramm** [comment]
@fg91 thanks, this should be fixed with v5.7.6
